CHANGELOG.md

```diff
@@ -13,6 +13,10 @@
 
 ## Master
 
+## 3.4.8
```

Comment:
please avoid version changes, I do that as I ship a new one and you'll just get unnecessary merge conflicts